interface-definitions/interfaces-wireless.xml.in

```diff
@@ -16,6 +16,7 @@
           </valueHelp>
         </properties>
         <children>
+          #include <include/interface-mirror.xml.i>
```

Comment:
Does this actually work? WIFI interfaces are kinda picky.

Reply:
Smoke test successful